src/resources/js/components/AccountCreation.vue

```diff
@@ -1,50 +1,124 @@
 <template>
-    <div class="container">
-            <h2> {{ greetings }} </h2>
-            <form class="form-group" @submit.prevent="onSubmit">
-            <div class="ui input">
-                <label for="fistname" > Username : </label>
-                <input id="name" v-model="firstname" required>
-            </div>
-
-            <div class="ui input">
-                <label for="lastname"> password : </label>
-                <input id="name" v-model="lastname" required>
-            </div>
-            <div class="ui input">
-                <label for="lastname"> password confirmation : </label>
-                <input id="name" v-model="lastname" required>
-            </div>
-
-            
-            <div class="ui primary button">
-                <input type="submit" value="Submit">  
-            </div>
-
-            
-        </form>
-            
-
-    </div>
+  <div id="app">
+  
+    <p class="title">PAUL</p>
+    <form @submit="register" method="post">
+      <div class="form-group">
+        <input type="text" class="input" name="username" v-model="input.username" placeholder="Nom d'utilisateur" />
+      </div>
+      <div class="form-group">
+        <input type="email" class="input" name="email" v-model="input.email" placeholder="email" />
+      </div>
+      <div class="form-group">
+        <input type="password" class="input" name="password" v-model="input.password" placeholder="Mot de passe" />
+      </div>
+      <div class="form-group">
+        <input type="password" class="input" name="passwordConfirm" v-model="input.passwordConfirm" placeholder="Confirmer mot de passe" />
+      </div>
+      <div class="form-group">
+        <button type="button" class="button" v-on:click="register()">Créer compte</button>
+      </div>
+      
+      <p v-if="isBlank"> Manque des informations</p>
+      <p v-if="isIncorect"> mot de passe différent</p>
+    </form>
+  </div>
 </template>
 
 <script>
+    import axios from 'axios'
+    import router from '../routes'
+
     export default {
-        data() {
-            return {
-                greetings: 'Please fill the followings',
-                firstname: null,
-                lastname: null,
+    name : 'Register',
+    data () {
+        return{
+          input: {
+              username: '',
+              email: '',
+              password: '',
+              passwordConfirm: ''
+          },
+      darkBlue: '#5F74E8',
```

Comment:
Theses attributes aren't in use, you can remove they

Reply:
Changed in bfdd2f5